config/initializers/address_lookup.rb

```diff
@@ -4,7 +4,7 @@
   config.address_facade_url     = "/address-service/v1/addresses/postcode"
 
   # 5 was causing many timeouts in development
-  config.timeout_in_seconds     = 10
-  config.address_facade_client_id =  Rails.application.secrets.address_facade_client_id
-  config.address_facade_key       =  Rails.application.secrets.address_facade_key
+  config.timeout_in_seconds       = Rails.application.secrets.address_facade_timeout
+  config.address_facade_client_id = Rails.application.secrets.address_facade_client_id
+  config.address_facade_key       = Rails.application.secrets.address_facade_key
```

Comment:
no suggestions, but have realised the engine is making assumptions here that the host app will have a secrets file with those settings - not a big issue for us as we control the host apps - will just need to make sure fo and bo and dummy all have that setting.